dist/templates/ovnkube-node.yaml.j2

```diff
@@ -140,11 +140,9 @@ spec:
         - mountPath: /etc/ovn/
           name: host-var-lib-ovs
           readOnly: true
-        {% if kind is defined and kind -%}
         - mountPath: /var/run/netns
           name: host-netns
           mountPropagation: Bidirectional
```

Comment:
not sure if mountProp is needed even in kind case as ovnkube node is just reading off this folder and not writing to it.